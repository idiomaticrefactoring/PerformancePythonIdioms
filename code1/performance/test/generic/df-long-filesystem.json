[{"filesystem":"/dev/mapper/VolGroup00-LogVol00","type":"ext3","1024_blocks":6030784,"used":1147932,"available":4571556,"mounted_on":"/","capacity_percent":21},{"filesystem":"proc","type":"proc","1024_blocks":0,"used":0,"available":0,"mounted_on":"/proc","capacity_percent":null},{"filesystem":"sysfs","type":"sysfs","1024_blocks":0,"used":0,"available":0,"mounted_on":"/sys","capacity_percent":null}]
