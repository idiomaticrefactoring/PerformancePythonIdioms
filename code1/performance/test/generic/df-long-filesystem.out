Filesystem    Type 1024-blocks      Used Available Capacity Mounted on
/dev/mapper/VolGroup00-LogVol00 ext3     6030784   1147932   4571556      21% /
proc          proc           0         0         0       -  /proc
sysfs        sysfs           0         0         0       -  /sys
